app/javascript/css/forms.scss

```diff
@@ -447,3 +447,25 @@ input::-webkit-datetime-edit-year-field:focus {
     background-color: var(--badge-current-bg);
     outline: none;
 }
+
+.admin-designation:after {
+  content: '⚙';
+  margin-left: 0.25ch;
+  color: var(--success-color);
+  font-weight: 600;
+  display: inline;
+}
+
+.remove-file {
+  color: var(--error-color);
+  cursor: pointer;
+  display: flex;
+
+  .dismiss-icon {
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 24px;
+    width: 24px;
+    background-image: url('data:image/svg+xml,<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M1 1L11 11M11 1L1 11" stroke="%23A21E1F" stroke-width="1.5" stroke-linecap="round"/></svg>');
```

Comment:
stroke color ideally would be coming from the current color rather than hard-coded